odoo/models.py

```diff
@@ -5322,8 +5351,9 @@ def recompute(self):
                     continue
                 vals = rec._convert_to_write(vals)
                 updates[frozendict(vals)].add(rec.id)
+            st_timeout = recs.env['ir.config_parameter'].sudo().get_param('compute_statement_timeout')
```

Comment:
Can you set a default value in case is not set like it was done at:

https://github.com/odoo/enterprise/blob/12.0/currency_rate_live/models/res_config_settings.py#L264-L269
```
        icp = self.env['ir.config_parameter'].sudo()
        token = icp.get_param('banxico_token')
        if not token:
            # https://www.banxico.org.mx/SieAPIRest/service/v1/token
            token = 'd03cdee20272f1edc5009a79375f1d942d94acac8348a33245c866831019fef4'  # noqa
            icp.set_param('banxico_token', token)
```

Reply:
@moylop260 

Regards.

Reply:
I preferred don't use default in order to disable entirely the `SET` queries

I mean, removing the parameter the method will "yield" early with 0 extra queries executed to SET PSQL parameters

In order to use the original way of Odoo